doc/api/async_hooks.md

```diff
@@ -1028,6 +1028,8 @@ added:
  - v12.17.0
 -->
 
+> Stability: 2 - Stable
```

Comment:
Please update the YAML section above with something like
```
 - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/37675
    description: AsyncLocalStorage is now Stable. Previously, it had been Experimental.
```
e.g. HTTP/2 did it that way.

Reply:
Good point. Done!